README.md

```diff
@@ -1,127 +1,187 @@
+[![Docs Status](https://app.readthedocs.org/projects/graphstorm/badge/?version=latest)](https://graphstorm.readthedocs.io/en/latest/)
+
+| [Documentation and Tutorial Site](https://graphstorm.readthedocs.io/en/latest/) | [GraphStorm Paper](https://arxiv.org/abs/2406.06022) |
+
+GraphStorm is an enterprise-grade graph machine learning (GML) framework designed for scalability and ease of use.
+It simplifies the development, training, and deployment of GML models on industry-scale graphs with billions of nodes and edges.
 
-GraphStorm is a graph machine learning (GML) framework for enterprise use cases.
-It simplifies the development, training and deployment of GML models for industry-scale graphs
-by providing scalable training and inference pipelines of Graph Machine Learning (GML) models
-for extremely large graphs (measured in billons of nodes and edges).
 GraphStorm provides a collection of built-in GML models and users can train a GML model
 with a single command without writing any code. To help develop SOTA models,
 GraphStorm provides a large collection of configurations for customizing model implementations
 and training pipelines to improve model performance. GraphStorm also provides a programming
 interface to train any custom GML model in a distributed manner. Users
 provide their own model implementations and use GraphStorm training pipeline to scale.
 
+## Key Features
+- Single-command GML model training
+- Distributed training on industry-scale graphs (billions of nodes/edges)
+- Built-in model collection
+- AWS integration out-of-the-box
+
+
 ![GraphStorm architecture](https://github.com/awslabs/graphstorm/blob/main/tutorial/graphstorm_arch.jpg?raw=true)
 
 ## Get Started
+
+
 ### Installation
-GraphStorm is compatible to Python 3.8+. It requires PyTorch 1.13+, DGL 1.0+ and transformers 4.3.0+. GraphStorm only supports DGL up to version 2.3.0.
 
-GraphStorm can be installed with pip and it can be used to train GNN models in a standalone mode. To run GraphStorm in a distributed environment, we recommend users to using [Docker](https://docs.docker.com/get-started/overview/) container to reduce environment setup efforts. A guideline to setup GraphStorm running environment can be found at [here](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-docker-environment) and a full instruction on how to setup distributed training can be found [here](https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/distributed/cluster.html). For quick installation, please refer to the following command:
-```
-pip install torchdata==0.9.0 pydantic
-#CPU
-pip install torch==2.3.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+GraphStorm is compatible with Python 3.8+. It requires PyTorch 1.13+, DGL 1.0+ and transformers 4.3.0+.
```

Comment:
Let's put a link to https://graphstorm.readthedocs.io/en/v0.4.1/#getting-started here.